test/e2e_clutz_dts_test.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+///<reference types="jasmine"/>
```

Comment:
In every instance but this one you had a blank line after this import

Reply:
if the formatter doesn't format it then how can a human possibly do it? :man_shrugging: